.github/workflows/ci.yaml

```diff
@@ -22,15 +22,18 @@ jobs:
   build:
     needs: ci
     runs-on: ubuntu-latest
+    env:
+      TRAVIS_BRANCH: ${{ github.head_ref }}
+      TRAVIS_PULL_REQUEST: ${{ github.event.number || 'false' }}
     steps:
     - uses: actions/checkout@v2
+    - name: Set ENV variables
+      run: |
+        echo "::set-env name=TRAVIS_BRANCH::${TRAVIS_BRANCH:-$(echo $GITHUB_REF | cut -d '/' -f 3)}"
```

Comment:
This is annoying, but apparently it's the only way to get the actual branch name.